lib/operators/input.coffee

```diff
@@ -24,6 +24,8 @@ class Insert extends Operator
           @typedText,
           normalizeLineEndings: true
         )
+        cursor = @editor.getLastCursor()
+        cursor.moveLeft() unless cursor.isAtBeginningOfLine()
```

Comment:
Got it. I didn't realize that repeating insert operations still only worked on the last cursor.